src/lib.rs

```diff
@@ -456,6 +456,12 @@ pub trait ConversionFactor<V>:
     fn value(self) -> V;
 }
 
+/// Helper trait to identify the zero value of a type at compile time.
+pub trait ConstZero {
```

Comment:
We should have at least a simple doc example. Perhaps it can be improved in some way?

```suggestion
/// Helper trait to identify the zero value of a type at compile time.
///
#[cfg_attr(all(feature = "si", feature = "f32"), doc = " ```rust")]
#[cfg_attr(not(all(feature = "si", feature = "f32")), doc = " ```rust,ignore")]
/// # use uom::si::f32::Length;
/// use uom::ConstZero;
/// 
/// let zero = Length::ZERO;
/// ```
pub trait ConstZero {
```

Reply:
Committed a slightly extended version that constructs a new constant using `ZERO` constants.